### PR TITLE
Server

### PR DESCRIPTION
**Pull request by daemon**
Exposes model via REST API. Also added example client.

### CODE REVIEW

File: server.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+            wav_data: The WAVE to label
+
+        Returns:
+            A (most likely label, probability) tuple
+        """
+        output = self.sess.graph.get_tensor_by_name("labels_softmax:0")
+        predictions, = self.sess.run(output, {"wav_data:0": wav_data})
+        return (self.labels[np.argmax(predictions)], max(predictions))
+
+def json_in(f):
```
Comment (tuzhucheng) [inline server.py:34]: 4 spaces for indent

File: utils/client.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+            print(self.contains_command(b''.join(buf)))
+            buf[0] = buf[1]
+            buf[1] = stream.read(chunk_size)
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--endpoint",
+        type=str,
+        default="http://127.0.0.1:16888",
```
Comment (tuzhucheng) [inline utils/client.py:33]: Consider changing this to `0.0.0.0`? Not sure if with `127.0.0.1` the client can successfully reach the server on a separate machine.

**Review by tuzhucheng — APPROVED** [review]
Curious about choice of CherryPy over Flask.

LGTM after comments addressed.

File: server.py (hunk 1 above)
Comment (daemon) [reply server.py:34]: Oh oops, I copied this from one of my older projects...

File: utils/client.py (hunk 2 above)
Comment (daemon) [reply utils/client.py:33]: I'm currently port forwarding dragon00:16888 to my local machine using an SSH tunnel

**Comment by daemon** [comment]
None in particular, I'm just more familiar with CherryPy.

EDIT: Actually Flask looks more clean for this kind of stuff, I'll switch later.

**Pull request merged**